docs/02-app/01-building-your-application/06-optimizing/02-videos.mdx

```diff
@@ -0,0 +1,281 @@
+
+### Adding Subtitles to Your Video
+
+If you have subtitles for your video, you can easily add them using the `<track>` element inside your `<video>` tag. You can fetch the subtitle file from Vercel Blob in a similar way as the video file. Here's how you can update the VideoComponent to include subtitles
+
+```jsx filename="app/page.jsx"
+async function VideoComponent({ fileName }) {
+  const {blobs} = await list({
+    prefix: fileName,
+    limit: 2
+  });
+  const { url } = blobs[0];
+  const { url: captionsUrl } = blobs[1];
+
+  return (
+    <video controls preload="none" aria-label="Video player">
+      <source src={url} type="video/mp4">
+      <track
+        src={captionsUrl}
+        kind="subtitles"
+        srclang="en"
+        label="English">
+      Your browser does not support the video tag.
+    </video>
+  );
+};
+```
+
+By following this approach, you can effectively self-serve and integrate videos into your Next.js applications, offering a high degree of control and customization over your video content.
+
+## Video Optimization Techniques
+
+Enhancing performance and user experience involves strategic optimization of video content. Key techniques include video compression, selecting the appropriate format, and implementing adaptive streaming. Here’s a concise overview of the key techniques:
+
+1. **Understanding Video Formats and Codecs**:
+   - Select formats like MP4 (widely compatible, efficient), WebM (open-source, optimized for web), and AVI (high quality, larger file size) based on compatibility and quality needs.
+   - Choose codecs such as H.264 (high compression, quality) or VP9 (optimized for web streaming) for the best balance between quality and file size.
+2. **Video Compression**:
+   - Use tools such as [FFmpeg](https://www.ffmpeg.org/) to compress videos, reducing file size while maintaining acceptable quality.
+   - Experiment with compression settings to find the ideal balance between quality and file size, considering the intended audience and platform.
+3. **Resolution and Bitrate Adjustment**:
+   - Adjust resolution and bitrate based on the target platform: lower for mobile and higher for desktop or TV viewing.
+   - Lower bitrate and resolution for background videos to save bandwidth without noticeably affecting visual quality.
+4. **Content Delivery Networks (CDNs)**:
+   - Use a CDN to distribute video content, reducing latency and improving load times globally.
+   - CDNs help manage large traffic and enhance user experience by delivering content from the nearest server.
+5. **Lazy Loading of Videos**:
+   - Implement lazy loading to defer loading of videos until they are in view, improving initial page load times.
+
+By incorporating these techniques, you can significantly enhance the performance and accessibility of videos, leading to a better user experience and increased user engagement.
```

Comment:
While these are video best practices to know about, they don't directly relate to Next.js. So I would say if it doesn't lead them to take action or we don't have good resources to link to where the user can learn more, then we can cut. 

Additionally, lazy loading contracts with streaming. You don't need to lazy load with Server Components and Suspense. 

Maybe you can rename this section "Resources", and keep the resources listed below this text.